api/src/rest/views/entity.py

```diff
@@ -21,6 +21,7 @@ class EntityCAView(generics.ListAPIView):
     queryset = models.Entity.objects.all()
     serializer_class = serializers.EntityBasicSerializer
 
+
 class EntityByCN(generics.RetrieveAPIView):
     serializer_class = serializers.EntityBasicSerializer
 
```

Comment:
Dla EntityCAAuthsView znów chyba da się filtrem zrobić, zakładając że lookup_field to `id`:

entity = filters.CharFilter(label="Entity", field_name='entity__id')

W zapytaniu będzie ?entity=2

Reply:
Przerabiam views żeby korzystały z viewsetów/routerów + filtry/paginacja/search, to się chyba przyda przy własnym frontendzie.